generators/new-wp-plugin/index.js

```diff
@@ -92,11 +91,33 @@ module.exports = generators.Base.extend({
       regex: 'lean-p',
       replacement: this.name
     }, settings) );
+    console.log( chalk.green('Completed!\n') );
   },
 
   install: function() {
     process.chdir( this.PLUGIN_FOLDER );
 
-    this.spawnCommandSync('composer', ['update']);
+    if ( fs.existsSync('./vendor') ) {
+      console.log( chalk.red('Composer has alredy been installed\n') );
+    } else {
+      this.spawnCommandSync('composer', ['install']);
```

Comment:
Please refer to the composer docs, and the issues that are created on this repo associated with the scripts that composer fires.


> On Jun 1, 2016, at 8:23 AM, Adam Fenton <notifications@github.com> wrote:
> 
> In generators/new-wp-plugin/index.js <https://github.com/moxie-lean/generators/pull/31#discussion_r65358377>:
> 
> >    },
> >  
> >    install: function() {
> >      process.chdir( this.PLUGIN_FOLDER );
> >  
> > -    this.spawnCommandSync('composer', ['update']);
> > +    if ( fs.existsSync('./vendor') ) {
> > +      console.log( chalk.red('Composer has alredy been installed\n') );
> > +    } else {
> > +      this.spawnCommandSync('composer', ['install']);
> Why do we need this? I think composer install is enough.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/moxie-lean/generators/pull/31/files/71c9ab02e1744c55d6781a72d662c355f667938d#r65358377>, or mute the thread <https://github.com/notifications/unsubscribe/ADvViSd6JDPGOSQlBAMO2KHtgaFkef5Kks5qHYfNgaJpZM4InFiZ>.
>